### PR TITLE
Tidy LimitRange concept

### PR DESCRIPTION
**Pull request by sftim**
Minor tidying for https://kubernetes.io/docs/concepts/policy/limit-range/ [[preview](https://deploy-preview-20249--kubernetes-io-master-staging.netlify.com/docs/concepts/policy/limit-range/)]

/kind cleanup

### CODE REVIEW

**Comment by ysyukr** [comment]
@sftim PR (#20221 ) that changes the link in the LimitRange document I opened, so how about combining it with this PR?

File: content/en/docs/concepts/policy/limit-range.md [hunk 1]
```diff
@@ -48,27 +48,28 @@ The name of a LimitRange object must be a valid
...
-- See a [detailed example on configuring quota per namespace](/docs/tasks/administer-cluster/quota-memory-cpu-namespace/).
 
 {{% /capture %}}
 
 {{% capture whatsnext %}}
 
-See [LimitRanger design doc](https://git.k8s.io/community/contributors/design-proposals/resource-management/admission_control_limit_range.md) for more information.
+Refer to the [LimitRanger design document](https://git.k8s.io/community/contributors/design-proposals/resource-management/admission_control_limit_range.md) for more information.
+
+<span id="examples">For examples on using limits</span>, see:
```
Comment (kbhawkey) [inline content/en/docs/concepts/policy/limit-range.md:66]: @sftim ,
Is there a good reason to include HTML?

Comment (sftim) [reply content/en/docs/concepts/policy/limit-range.md:66]: I want to preserve the incoming links to `#examples` and don't know how to do that in pure Markdown.

Comment (kbhawkey) [reply content/en/docs/concepts/policy/limit-range.md:66]: This may not render with new versions of Hugo.

**Comment by sftim** [comment]
Hello Prow/Tide

**Pull request merged**